code/obj/item/ai_modules.dm

```diff
@@ -335,6 +336,39 @@ AI MODULES
 		//AI.eyecam.name = lawTarget //not sure if we need?
 
 
+/********************* EXPERIMENTAL LAWS *********************/
+//note: these 'experimental' laws are spawned in via random_item_spawner instead of spawning every round
+//so, any given one of these will only have a chance of spawning
+//[IMPORTANT] The pool for the experimental laws starts on line 810 in file /code/obj/random_spawners.dm. UPDATE THAT TO INCLUDE ANY NEW LAWS SO THEY CAN SPAWN!!!
+//also if you notice that there's like 5 or 6 experimental modules in the spawn list, maybe wanna increase the max amount that can spawn?
```

Comment:
I think comments like this that couple the code to another file so tightly are probably a little bit of overkill. Things like referencing line numbers is bound to get stale at some point. For example, if another way of obtaining these modules was given, this comment would immediately be out of date.

The functionality of the modules is defined here, not how they're used, and I think these comments would be better served over in `/code/obj/random_spawners.dm`.

Reply:
Gotcha! I'll just refer to the files they're present in, then!

Reply:
There is already a comment in the random_spawner file explaining their useage in a single comment - I presume that's sufficient? (Also the previous changes have already been made)